react/features/whiteboard/components/web/Whiteboard.tsx

```diff
@@ -118,6 +130,20 @@ const Whiteboard: () => JSX.Element = () => {
             {
                 isOpen && (
                     <div className = 'excalidraw-wrapper'>
+                        {/*
+                          * Excalidraw renders a few lvl 2 headings. This is
+                          * quite fortunate, because we actually use lvl 1
+                          * headings to mark the big sections of our app. So make
+                          * sure to mark the Excalidraw context with a lvl 1
+                          * heading before showing the whiteboard.
+                          */
+                            <span
```

Comment:
Same question as above.

Reply:
What's your question haha? Not sure I understand.

Reply:
It was the one regarding spans and divs, but we clarified.